frontend/src/app/testResults/viewResults/DownloadResultsCsvModal.tsx

```diff
@@ -70,8 +70,8 @@ export const DownloadResultsCsvModal = ({
   const handleComplete = (data: GetFacilityResultsForCsvWithCountQuery) => {
     if (data?.testResultsPage?.content) {
       const csvResults = parseDataForCSV(
-        hivEnabled,
-        data.testResultsPage.content
+        data.testResultsPage.content,
+        disabledFeatureDiseaseList
```

Comment:
Could you help me understand the advantages of using this separate `disabledFeatureDiseaseList` pattern across this method / everywhere else? Coming from the outside, it seems to me that if we just used the enabled version, we wouldn't need multiple methods in the util file that checks for whether the inverse of the feature flags and maybe have a simpler set of checks as a result. 

Do let me know if I'm missing something!

Reply:
This is mostly related to how `parseDataForCsv` previously used an `hivEnabled` parameter to determine whether or not to add that column so in this case I made it into a list of disabled diseases. Inside the `parseDataForCsv` we could certainly invert this to use the `supportedDiseaseList` and check for the existence of HIV and Syphilis to determine whether to add those columns, but a downside is that for the unit tests for `parseDataForCsv` we then need to explicitly provide all the supported diseases. Whenever we update these parts of the code in the future for adding additional diseases, it's likely going to be in the context of the new disease having an enabled/disabled feature flag so with that in mind I felt it made more sense to refer to those disabled diseases explicitly rather than refer to all diseases.

Comparing 

```
expect(
      parseDataForCSV(data, [
        MULTIPLEX_DISEASES.HIV,
        MULTIPLEX_DISEASES.SYPHILIS,
      ])
    ).toEqual(resultNoSTD);
```

to 

```
expect(
      parseDataForCSV(data, [
        MULTIPLEX_DISEASES.COVID_19,
        MULTIPLEX_DISEASES.FLU_A,
        MULTIPLEX_DISEASES.FLU_B,
        MULTIPLEX_DISEASES.RSV,
        MULTIPLEX_DISEASES.HIV,
        MULTIPLEX_DISEASES.SYPHILIS,
        ...all the other diseases in the future
      ])
    ).toEqual(resultNoSTD);
```